client/configuration_variable.go

```diff
@@ -110,15 +120,12 @@ func (client *ApiClient) ConfigurationVariablesByScope(scope Scope, scopeId stri
 		return []ConfigurationVariable{}, err
 	}
 
+	// The API returns global and template scopes for environment (and other) scopes. Filter them out.
```

Comment:
global and blueprint filtering is done in the API.

Reply:
if not global - remove global...
if not global and not template,  remove template as well.